csharp/src/Apache.Arrow/Apache.Arrow.csproj

```diff
@@ -1,20 +1,24 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.3;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
This may be making it a bit harder to merge to master later, but I think it is reasonable.

Reply:
Yeah, assuming they want to go with the approach of having the decompressors implemented outside of the Arrow project this will be fine. If we want them implemented in the project then we could possibly disable the implementations for netstandard1.3 with preprocessor directives and add netstandard2.0 as a new target.